python/nano/src/bigdl/nano/pytorch/strategies/ddp_spawn.py

```diff
@@ -230,18 +233,10 @@ def setup(self, trainer: "pl.Trainer") -> None:
         # which is used by ipex==1.9, so we move this line here
         self.model_to_device()
 
-    def configure_ddp(self):
```

Comment:
Could you explain a little bit why changing here?

Reply:
We need tto change the creation of `DistributedDataParallel`

in the default implementation of pl 1.6, the trainer will call `configure_ddp`, and `configure_ddp` will call `_setup_model` to create a DistributedDataParallel, but LightningLite will call `_setup_model` directly to create a DistributedDataParallel.

we choose to override the `configure_ddp` and create DistributedDataParallel directly in it before, but this method makes no effect when using LightningLite, so we choose to override the `_setup_model`, which will be used in both cases.